tests/app/dao/test_templates_dao.py

```diff
@@ -96,6 +96,28 @@ def test_update_template(sample_service, sample_user):
     assert dao_get_all_templates_for_service(sample_service.id)[0].name == 'new name'
 
 
+def test_update_template_with_reply_to_email(sample_service, sample_user):
+    data = {
+        'name': 'Sample Template',
+        'template_type': "sms",
+        'content': "Template content",
+        'service': sample_service,
+        'created_by': sample_user,
+        'reply_to_email': "some@one.com"
+    }
+    template = Template(**data)
+    dao_create_template(template)
+    created = dao_get_all_templates_for_service(sample_service.id)[0]
+    assert created.name == 'Sample Template'
+    assert created.reply_to_email == "some@one.com"
+
+    created.name = 'new name'
+    created.reply_to_email = "testing@change.com"
+    dao_update_template(created)
+    assert dao_get_all_templates_for_service(sample_service.id)[0].name == 'new name'
+    assert dao_get_all_templates_for_service(sample_service.id)[0].reply_to_email == "testing@change.com"
+
+
```

Comment:
I recommend writing a test that updates multiple fields at once.

Reply:
This should be able to be tested after all the updates are completed and will be located with any other API tests and maybe not specifically the DAO tests.

Reply:
Test was not needed and has been removed. Multiple update functionality is part of the endpoint now as I misunderstood what was going on earlier in the checks.